authoringAssistant/settings.json

```diff
@@ -11,6 +11,6 @@
   "ollama-port": "11434",
   "log-path": "logs/",
   "fluid-compiler-path": "./",
-  "fluid-temp-file": "example/llmTest",
+  "fluid-temp-file": "website/Ar6Wg1SPM/fluid/llmTest",
```

Comment:
I’ll accept the PR anyway but perhaps it makes more sense to have the temp file be unrelated to the website?